orttraining/orttraining/python/experimental/orttrainer.py

```diff
@@ -581,6 +581,9 @@ def _create_ort_training_session(self):
         ort_parameters.weights_to_train = trainable_params
         ort_parameters.optimizer_attributes_map = optimizer_attributes_map
         ort_parameters.optimizer_int_attributes_map = optimizer_int_attributes_map
+        ort_parameters.max_num_pre_training_graph_transformation_steps = \
```

Comment:
Is it possible to add a unit test for each new flag? How do we check whether it is working?

Reply:
added a test, let me know if you had something else in mind